### ISSUE TITLE
`INFO` seems to be throwing error against `master` branch of `RedisTimeSeries`

### ISSUE THREAD

**Issue by dpipemazo** [RedisTimeSeries/redistimeseries-py#60]
Just upgraded RedisTimeSeries to commit `e8030d14e7b8e186518b1b551cfa3cde261bf3c3` and am now seeing this when calling `client.info()` with `redistimeseries-py`:

```
    data = self._mclient.info(key)
  File "/opt/venv/lib/python3.7/site-packages/redistimeseries/client.py", line 301, in info
    return self.execute_command(self.INFO_CMD, key)
  File "/opt/venv/lib/python3.7/site-packages/redis-3.4.1-py3.7.egg/redis/client.py", line 901, in execute_command
  File "/opt/venv/lib/python3.7/site-packages/redis-3.4.1-py3.7.egg/redis/client.py", line 921, in parse_response
  File "/opt/venv/lib/python3.7/site-packages/redistimeseries/client.py", line 29, in __init__
    self.maxSamplesPerChunk = response['maxSamplesPerChunk']
KeyError: 'maxSamplesPerChunk'
```

Ran `INFO` on a key in my RedisTimeSeries and am now seeing this response:
```
 1) totalSamples
 2) (integer) 57
 3) memoryUsage
 4) (integer) 4474
 5) firstTimestamp
 6) (integer) 1598402024391
 7) lastTimestamp
 8) (integer) 1598402629741
 9) retentionTime
10) (integer) 86400000
11) chunkCount
12) (integer) 1
13) chunkSize
14) (integer) 4096
15) labels
16) 1) 1) "element"
       2) "monitor"
    2) 1) "type"
       2) "process"
    3) 1) "container"
       2) "x86_64"
    4) 1) "device"
       2) "default"
    5) 1) "language"
       2) "Python"
    6) 1) "version"
       2) "1.7.2"
    7) 1) "level"
       2) "INFO"
    8) 1) "subtype0"
       2) "3912:kworker/9:2"
    9) 1) "subtype1"
       2) "cpu_user"
17) sourceKey
18) (nil)
19) rules
20) (empty array)
```

Perhaps this response has changed? This was previously working OK against release `1.2.7`

**Comment by filipecosta90** [comment]
hi there @dpipemazo you're totally right ( the reply changed ) and this issue should be fixed by https://github.com/RedisTimeSeries/redistimeseries-py/pull/59. The fix will be backwards compatible, so it should work both on 1.2 and 1.4 :)

**Comment by filipecosta90** [comment]
Hi there @dpipemazo this fixed has been merged on #59 and tested for both v1.2 and v1.4. Closing the issue and thank you for the fast report :)

**Comment by dpipemazo** [comment]
Thanks for the fix! Verified on my end, apologies for the late feedback.
